.github/workflows/_link_check.yml

```
@@ -18,7 +18,9 @@ jobs:
       script: |
         ./scripts/lint_urls.sh $(
           [ "${{ github.event_name }}" = "pull_request" ] \
-            && git diff --name-only ${{ github.event.pull_request.base.sha }} ${{ github.event.pull_request.head.sha }}
+            && git diff --name-only ${{ github.event.pull_request.base.sha }} ${{ github.event.pull_request.head.sha }} \
+          || [ "${{ github.event_name }}" = "push" ] \
+            && git diff --name-only ${{ github.event.before }} ${{ github.sha }}
```

Do you need to do this?

```suggestion
            && git diff --name-only ${{ github.event.pull_request.base.sha }} ${{ github.event.pull_request.head.sha }} \
          || ([ "${{ github.event_name }}" = "push" ] \
            && git diff --name-only ${{ github.event.before }} ${{ github.sha }})
```